src/service/date-util.service.ts

```diff
@@ -65,9 +65,9 @@ export class DateUtils {
     dateformat () {
         return this.pattern;
     }
-    //TODO Change this method when moving from datetime-local input to NgbDatePicker
+    // TODO Change this method when moving from datetime-local input to NgbDatePicker
     toDate(date: any): Date {
```

Comment:
Changing `date` to `string` implies that `dateParts` is of type `string[]`
Thus we have to use `parseInt(string, radix)` everytime we use `dateParts[i]`.

Should I go back to `any` or do I make the change ?